src/tox/tox_env/python/virtual_env/api.py

```diff
@@ -109,6 +109,9 @@ def session(self) -> Session:
 
     def virtualenv_env_vars(self) -> dict[str, str]:
         env = self.environment_variables.copy()
+        # we want to use PATH from environment rather than any PATH
+        # modifications in set_env to find the python executable
+        env["PATH"] = os.environ.get("PATH", "")
```

Comment:
This is not virtualenv specific logic, so has no place to live here.

Reply:
This is within the method `virtualenv_env_vars` which is used to construct the environment variables used when constructing the virtual env.  Specifically, these are not the environment variables used when running commands within the virtual environment.

The PATH set here is used to find the python executable to be used to construct the virtual environment.

As such, it seems virtualenv-specific to me.

Reply:
Wouldn't `base_python` be the better way to specify a custom interpreter?

Reply:
That would be short-sighted because those custom python interpreters could not be discovered by virtualenv within the commands, e.g. imagine your test tries creating a new virtual environment nested. In general is better to use PATH as that makes it more nestable.